src/arrayMethodSort.js

```diff
@@ -1,11 +1,36 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
+function defaultSort(a, b) {
+  if (`${a}` > `${b}`) {
```

Comment:
why do you need to use interpolation here?

Reply:
It has to be strings